grid.cpp

```diff
@@ -320,6 +320,32 @@ void initSpatialCellCoordinates(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geomet
    }
 }
 
+/*
+Record for each cell which processes own one or more of its face neighbors
+ */
+void setFaceNeighborRanks( dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid ) {
+
+   const auto& cells = mpiGrid.get_cells();
+
+   for (const auto& cellid : cells) {
```

Comment:
Can be threaded (but is it too short?).

Reply:
It's over all local cells so I suppose it can be fairly large. Are the DCCRG calls thread-safe?

Reply:
They are as long as we call things for only one cell, afaik.

Reply:
But maybe not a top priority, add a // TODO or FIXME?

Reply:
Done

Reply:
Yep, I just don't want to introduce a new thing going broken! :)

Reply:
And it's being profiled so we'll see if it kills us.